src/Supply/Application/Service/InventoryImporter.php

```diff
@@ -61,6 +62,9 @@ public function import(string $host): void
             $campaigns = $this->client->fetchAllInventory($host);
         } catch (EmptyInventoryException $exception) {
             return;
+        } catch (UnexpectedClientResponseException $exception) {
+            // we can add to a log file this information, not now
```

Comment:
Comments like this could have TODO keyword. It will be easy to find them across code.

Reply:
There is a dedicated ticket: https://adshares.atlassian.net/browse/SRV-44. I hope that it's enough to not forget about it ;-)